toolkit/recordreplay/moz.build

```diff
@@ -10,7 +10,7 @@ SOURCES += [
     'HashTable.cpp',
     'JSControl.cpp',
     'ProcessRecordReplay.cpp',
-    'RecordReplayDriver.cpp',
+    # 'RecordReplayDriver.cpp',
```

Comment:
That said, I don't think we have a released version of the Windows driver yet, so I guess that script will fail for now.

Reply:
Oh. I think I commented that out when I was trying to use `mach build` instead of `build.js`. I'll uncomment it locally and see if it builds as we'd expect with `build.js`.

Reply:
Welp, uncommented that line and it compiled, so this change is unnecessary. The most recent commit I pushed undid it.